### PR TITLE
Add spine-ts runtime

### PR DESCRIPTION
**Pull request by NathanSweet**

### CODE REVIEW

File: spine-ts/webgl/src/Matrix4.ts [hunk 1]
```diff
@@ -0,0 +1,305 @@
...
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+ * PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES, BUSINESS INTERRUPTION, OR LOSS OF
+ * USE, DATA, OR PROFITS) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER
+ * IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ *****************************************************************************/
+
+module spine.webgl {
+	export let M00 = 0;
```
Comment (NathanSweet) [inline spine-ts/webgl/src/Matrix4.ts]: Should these guys have a type? const?

File: spine-ts/core/src/SkeletonBounds.ts [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+ * PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES, BUSINESS INTERRUPTION, OR LOSS OF
+ * USE, DATA, OR PROFITS) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER
+ * IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ *****************************************************************************/
+
+module spine {
+	export class SkeletonBounds {
+		minX = 0; minY = 0; maxX = 0; maxY = 0;
```
Comment (NathanSweet) [inline spine-ts/core/src/SkeletonBounds.ts]: I have a feeling these should be `minx: number;` etc? Do next couple lines need types? Can't find anywhere that TS infers field type from assignment in the declaration.

File: spine-ts/webgl/src/Matrix4.ts (hunk 1 above)
Comment (badlogic) [inline spine-ts/webgl/src/Matrix4.ts]: Made const, type is inferred from initializer.

File: spine-ts/core/src/SkeletonBounds.ts (hunk 2 above)
Comment (badlogic) [inline spine-ts/core/src/SkeletonBounds.ts]: TS infers field type from initializer.

http://sht.tl/udWkU0

https://www.typescriptlang.org/docs/handbook/type-inference.html

**Pull request merged**